### PR TITLE
Zowe Server Install Wizard

### PR DESCRIPTION
**Pull request by lparadkar-rocket**
Please take a look at both the versions and provide suggestions.

Describe your pull request here:
Working on Zowe Server Install Wizard with Andrew.

List the file(s) included in this PR:
test-zen-doc.md
test-zen-doc_Sean.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Review by 1000TurquoisePogs — APPROVED** [review]
I think it looks good, but here's my wishlist for further improvements:

1) SMPE or PAX pages of the existing documentation don't talk about the install wizard yet do they? Can we shout it out briefly in those sections so that people are aware of their options?

2) I wish I had a better suggestion, but its still a little strange to me that we have "Installing via SMPE" and "Installing via PAX" but the install wizard section in the sidebar isnt "Installing via..." it's just "overview".

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+* [Troubleshooting](#troubleshooting)
+  * [Failure to establish a TLS connection](#failure-to-establish-a-tls-connection)
+  * [Unable to continue with Wizard installation](#unable-to-continue-with-wizard-installation)
+
+
+## Benefits of Wizard installation
+
+- Reduces the need for YAML editing in z/OS Unix by handling these operations for you based on inputs to the prompts presented in the Wizard UI.
+
+- Inputs to the prompts are validated, so that invalid input or typos entered in any commands are prevented.
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:23]: entered in most commands*

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+### Setup Wizard connection to z/OS
+
+Field name| Description                
+---|---
+Host      |Value for the target z/OS system for Zowe Installation. For example, `mainframe.yourcompany.com`
+FTP Port  |The FTP Port number for internal use. The default port is 21. If not specified, the wizard uses the default port.
+User Name |Your z/OS username.
+Password  |Your z/OS password.
+
+3. Select Use FTP over TLS (Highly Recommended). Provide details as described in the following table:
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:65]: Select "Use FTP with TLS." (reword)

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        
+Field name | Description
+---|---
+Min TLS    |Select the minimum TLS version to accept the certificate from the server.
+Max TLS    |Select the maximum TLS version to accept the certificate from the server.
+4. Select **Accept all certificates**.
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:71]: Wrong. It is not recommended to select "Accept all certificates". Selecting "Accept all certificates" will disable certificate verificiation

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+5. Click **VALIDATE CREDENTIALS**.
+6. Click **Continue**.
+7. In the **Before you start** window, review the instructions. 
+8. In the **Job statement** field, customize the job statement if needed and click **SAVE AND VALIDATE**.
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:75]: *customize the job statement (or use default)

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+9. In the **Planning window**, provide details for z/OS Unix locations, identifiers, and z/OSMF details.
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:76]: *provide details for z/OS Unix locations (including Node and Java), identifiers, and z/OSMF details.

*Further down the page, provide [...]

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+## Choosing the Server Installation Type
+
+1. In the **Installation Type** window, select one of the following three installation types in the Wizard:
+
+    <details>
+    <summary>Download Zowe convenience build PAX from internet</summary>
+
+    Download the latest Zowe convenience build in .PAX format from [zowe.org](https://zowe.org) using the wizard without visiting the website.  <br />       1. Click **License Agreement**. <br />2. On the End User License Agreement for Zowe page, click **AGREE**.
+    </details>
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:95]: Then --> Continue to "Unpax"
Then --> "Begin Download"

Then wait for statuses to finish. After they finish, move onto Step 2

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+|Certificates Option 3|Zowe creates a keyring and populates the keyring by importing PKCS12 content from a dataset that you specify.|RACF: [ZWEIKRR3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRR3)<br /><br />TSS: [ZWEIKRT3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRT3)<br /><br />ACF2: [ZWEIKRA3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRA3)|
+|STC (NYI)|**Purpose**:<br /> ZWESLSTC is the job for running Zowe's webservers. ZWESISTC runs the APF authorized cross-memory server. The ZWESASTC job is started by ZWESISTC on an as-needed basis.<br /><br />**Action**:<br /> Copy the members ZWESLSTC, ZWESISTC, and ZWESASTC into your desired PROCLIB. If the job names are customized, modify the job name YAML values in `zowe.setup.security.stcs`|[ZWEISTC](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEISTC)|
```
Comment (1000TurquoisePogs) [inline docs/user-guide/install-zowe-server-install-wizard.md:161]: This is no longer (NYI).
It's implemented.

**Review by DivergentEuropeans — APPROVED** [review]
Looks pretty good now

File: docs/user-guide/install-zowe-server-install-wizard.md [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+    ` ~/Library/Logs/{app name}/main.log`
+    </details>
+
+    <details>
+    <summary>For Linux</summary>
+    
+    ` ~/.config/{app name}/logs/main.log`
+    </details>
+
+If you are still unsure how to proceed, make a backup of these log files, and then use the following procedure to remove the Wizard's cache.
```
Comment (DivergentEuropeans) [inline docs/user-guide/install-zowe-server-install-wizard.md:206]: "If you are still unsure how to proceed, make a backup of these log files, and then use the following procedure to remove the Wizard's cache." -->

"If you are still unsure how to proceed, (optional) make a backup of these log files, and then use the following procedure to remove the Wizard's cache."

**Review by janan07 — APPROVED** [review]
I did a bit more refactoring to bring this up to our Zowe doc standards. I think it's in good condition to publish.

**Pull request merged**